src/java/org/apache/cassandra/service/accord/AccordVerbHandler.java

```diff
@@ -52,45 +48,34 @@ public void doVerb(Message<T> message) throws IOException
         // TODO (desired): need a non-blocking way to inform CMS of an unknown epoch and add callback to it's receipt
 //        ClusterMetadataService.instance().maybeCatchup(message.epoch());
         logger.debug("Receiving {} from {}", message.payload, message.from());
+        Node.Id fromNodeId = endpointMapper.mappedId(message.from());
         T request = message.payload;
+
         long knownEpoch = request.knownEpoch();
         if (!node.topology().hasEpoch(knownEpoch))
         {
             node.configService().fetchTopologyForEpoch(knownEpoch);
             long waitForEpoch = request.waitForEpoch();
```

Comment:
Request really needs to document the difference between `knownEpoch` and `waitForEpoch`.

Reply:
To take that a step further... `knownEpoch` has a default implementation that is just `waitForEpoch` and it has no other implementations?

Reply:
Correct. Removing its uses from here, and will remove the method from Accord core in the sister PR as well. Simplified some related logic here, reducing nesting and a redundant call to `fetchTopologyForEpoch()`.